pages/contact/index.vue

```diff
@@ -210,4 +210,9 @@ ul li a:hover {
   background-color: rgba(255, 255, 255, 0.979);
   box-shadow: 0px 0px 18px rgba(116, 227, 235, 0.959);
 }
+.map-title{
```

Comment:
Use windicss classes wherever applicable to maintain consistency. We normally avoid css if there is a windi css class to do the same.

Reply:
Shifted `padding` and `border-radius` properties to windicss. Now using `map-title` class only to set the background of the text.